.github/ISSUE_TEMPLATE/kubernetes_update.md

```diff
@@ -22,7 +22,7 @@ List of items to do for upgrading to {version_1} {version_2} {version_3}
     - [ ] Minikube in `build/includes/minikube.mk` (Get the patch version [here](https://kubernetes.io/releases/) since minikube supports the latest Kubernetes release)
     - [ ] Kind in `build/includes/kind.mk` (Confirm {version_2} is supported and get the patch version [here](https://github.com/kubernetes-sigs/kind/releases))
 - [ ] Update the k8s image used in the helm [pre-delete-hook](https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/hooks/pre_delete_hook.yaml) to {version_2} (Get the patch version [here](https://hub.docker.com/r/lachlanevenson/k8s-kubectl))
```

Comment:
We need to update the link to:  https://hub.docker.com/layers/bitnami/kubectl/1.27.7/images/sha256-07726b35e759e4ac361ccca923c925959876e26b988e597528b48b65ef6d17f1?context=explore 

We use the `bitnami/kubectl` image.

Reply:
May I check out your branch and update it? I've never tried this before.

Reply:
Good catch - already done!